test.cc

```diff
@@ -38,6 +38,8 @@ SOFTWARE.
 DEFINE_string(cards_file, "cards.json", "Cards data file in JSON format.");
 DEFINE_int32(api_port, 8383, "Port to spawn CTFO RESTful server on.");
```

Comment:
We can reduce naming ambiguity by changing `CTFO RESTful server` into `CTFO API`.
What do you think, guys?